js/walutomat.js

```diff
@@ -211,13 +211,9 @@ module.exports = class walutomat extends Exchange {
         return status;
     }
 
-    makeUri (path, params) {
-        return '/api/' + this.version + '/' + this.url (path, params);
-    }
-
     sign (path, api = 'public', method = 'GET', params = {}, headers = undefined, body = undefined) {
         const url = this.urls['api'] + '/' + this.version + '/' + this.url (path, params);
-        const uri = this.makeUri (path, params);
+        const uri = '/api/' + this.version + '/' + this.url (path, params);
```

Comment:
It was intentionally in another function now /api/ will get changed to /$api/ in PHP

Reply:
Ah, you're right, forgot about it... That's a bug in the transpiler, no worries, I'll add a workaround )

Reply:
Done ;) I will look into a more elegant way to fix this quirk in the transpile.js a bit later...